plugins/modules/nomad_job.py

```diff
@@ -0,0 +1 @@
+./web_infrastructure/nomad_job.py
```

Comment:
This is not really a "web" thing, may I ask you to move it to `./clustering/nomad/nomad_job.py` once ready?

Reply:
ok